### DISCUSSION TITLE
3.2.4 may be broken in Filament

### DISCUSSION THREAD

**Discussion by panda-madness** [livewire/livewire#7438] [7. Bugs (archived)] [answered]
### Livewire version

3.2.4

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Upgrade your filament project to Livewire 3.2.4
2. Open any page, see a `ReferenceError: watch is not defined` error in the dev console.
3. Observe that some alpine functionality is broken (for me it was triggering a Filament modal window, the window will open but the display style would not toggle).

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by panda-madness** [comment]
This seems to be the problem snippet (line 3005 in livewire.js):
```js
  magic("watch", (el, { evaluateLater: evaluateLater2, cleanup: cleanup22 }) => (key, callback) => {
    let evaluate22 = evaluateLater2(key);
    let getter = () => {
      let value;
      evaluate22((i) => value = i);
      return value;
    };
    let unwatch = watch(getter, () => {
    });
    cleanup22(unwatch);
  });
```
The `watch` function in `let unwatch = ...` is undefined.

**Comment by dgironella** [comment]
Same problem.

https://github.com/livewire/livewire/discussions/7440

**Comment by tonypartridger** [comment]
Confirmed here 3.2.4 breaks watch. Rolling back to 3.2.3 resolves the watch function.

**Comment by panda-madness** [comment]
Sorry to directly ping you @calebporzio but maybe we can catch this while it's still hot.

**Reply by panda-madness** [reply]
Of course you push out a fix as I type this out. Legend!

**Comment by calebporzio** [comment, marked as answer]
Sorry, I broke the watcher in Alpine and didn't realized it was bundled with that release. Fixed it in 3.2.5. Thanks for the heads up everyone
